specifyweb/frontend/js_src/lib/components/FormSliders/RecordSelectorFromIds.tsx

```diff
@@ -183,6 +186,10 @@ export function RecordSelectorFromIds<SCHEMA extends AnySchema>({
         recordSetTable: schema.models.RecordSet.label,
       })
     : commonText.delete();
+
+  // Determine if in scope of new data entry and set title header
+  const [recordsetid] = useSearchParameter('recordsetid');
```

Comment:
Don't do this. It's a bad assumption that the query string argument would always be available when this component is called and that it is not used by any other component for a different reason.

I.e, imagine a feature where you can preview a record set in a dialog - your code won't work in that case.

Instead of getting record set id from a query string, it should be a prop. Or even better, find a way to not add new props - the more props there are the less maintainable the code and the more things can break